src/NServiceBus.AcceptanceTests/PubSub/When_publishing_an_event_with_the_subscriber_scaled_out.cs

```diff
@@ -21,19 +22,24 @@ public void Should_only_publish_one_event()
         {
             Scenario.Define<Context>()
                     .WithEndpoint<Publisher>(b =>
+                    {
+                        b.CustomConfig(configure => configure.SuppressOutdatedTimeoutDispatchWarning());
```

Comment:
the issue was fixed by changing name of test sqltransport, i forgot to remove it, thanks for catching!